src/firebase/remoteConfigValuesDefaults.ts

```diff
@@ -56,6 +56,7 @@ export const REMOTE_CONFIG_VALUES_DEFAULTS: Omit<
   showPriceChangeIndicatorInBalances: false,
   paymentDeepLinkHandler: PaymentDeepLinkHandler.Disabled,
   dappsWebViewEnabled: false,
+  dappListApiUrl: '',
```

Comment:
Just curious, why not setting `REMOTE_CONFIG_VALUES_DEFAULTS.dappListApiUrl` here?

Reply:
hum, i hope you don't mind - i didn't end up making this change. i think it's safer to leave this as an empty string so that it has the same behaviour has the default of `null` before. i think in reality it makes very little difference to the user what the initial value is anyway. it seems like we use a different value for alfajores and mainnet in remote config too, i'm not sure if it is intentional 🙈 i'm going to merge like this, lmk if you have strong opinions about this default value and i can always change it in a new PR :)